docker-compose/swe-idcc-idcf/config/swe-idcc-idcf-data-bridge-application.yml

```diff
@@ -0,0 +1,120 @@
+        time-validity: hourly
+        remote-file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_REN_TP_[0-9]{3}.zip"
+        source-directory: "/cgms"
+        sink-directory: "/CGMs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.ren-tp.properties
+      - file-type: RTE_EQ
+        file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_RTEFRANCE_EQ_[0-9]{3}.xml
+        time-validity: hourly
+        remote-file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_RTEFRANCE_EQ_[0-9]{3}.zip"
+        source-directory: "/cgms"
+        sink-directory: "/CGMs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.rte-eq.properties
+      - file-type: RTE_SSH
+        file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_.*-RTEFRANCE_SSH_[0-9]{3}.xml
+        time-validity: hourly
+        remote-file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_.*-RTEFRANCE_SSH_[0-9]{3}.zip"
+        source-directory: "/cgms"
+        sink-directory: "/CGMs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.rte-ssh.properties
+      - file-type: RTE_TP
+        file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_RTEFRANCE_TP_[0-9]{3}.xml
+        time-validity: hourly
+        remote-file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})T(?<hour>[0-9]{2})(?<minute>[0-9]{2})Z_ID_RTEFRANCE_TP_[0-9]{3}.zip"
+        source-directory: "/cgms"
+        sink-directory: "/CGMs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.rte-tp.properties
+      - file-type: CRAC
+        file-regex: 10VCORSWEPR-ENDE_18V0000000005KUU_SWE-CRAC-B15-A48-F011_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})-[0-9]{3}.xml
+        time-validity: daily
+        remote-file-regex: ".*"
+        source-directory: "/cracs"
+        sink-directory: "/CRACs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.crac.properties
+      - file-type: GLSK
+        file-regex: 10VCORSWEPR-ENDE_18V0000000005KUU_SWE-GLSK-B22-A48-F008_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})-[0-9]{3}.xml
+        time-validity: daily
+        remote-file-regex: ".*"
+        source-directory: "/glsks"
+        sink-directory: "/GLSKs"
+        file-list-persistence-file: /tmp/gridcapa/ftp-metadata-store.glsk.properties
+  sources:
+    ftp:
+      active: true
+      host: ftp-server
+      port: 21
+      username: gridcapa
+      password: gridcapa
+      base-directory: /data/gridcapa/swe/idcf
+      polling-delay-in-ms: 5000
+      max-messages-per-poll: 10
```

Comment:
I think "max-pool-size: 2" is missing here

Reply:
thanks I missed that one